server/src/controller/group.controller.ts

```diff
@@ -31,6 +39,16 @@ const createGroup = async (req: Request, res: Response, next: NextFunction) => {
     newRelation.lastAccessTime = now;
     await groupMemberRepository.save(newRelation);
 
+    const newMeetingChannel = new MeetingChannel();
+    newMeetingChannel.group = newGroup;
+    newMeetingChannel.name = 'default';
+    await meetingChannelRepository.save(newMeetingChannel);
```

Comment:
```suggestion
    await meetingChannelRepository.insert({ group: newGroup, name: 'default' });
```
이런식으로 하는게 더 깔끔할 것 같음!
회원가입도 save 써서 했는데 고쳐야 할 것 같아!
만약에 객체 생성해야 한다고 하면
```
const newMeetingChannel = new MeetingChannel({ group: newGroup, name: 'default' });
```
이렇게 생성자 호출할때 넣어주는게 가독성도 좋고 심플할듯?
내가 처음에 잘못해서 고칠게 많아지네 쏘리 ㅠㅠ